python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -389,7 +394,7 @@ class NearestNeighbors(Base,
     def get_param_names(self):
         return super().get_param_names() + \
             ["n_neighbors", "algorithm", "metric",
-                "p", "metric_params", "algo_params"]
+                "p", "metric_params", "algo_params", "n_jobs"]
```

Comment:
```suggestion
                "p", "metric_params", "algo_params"]
```
`n_jobs` is not really a hyperparameter fit for this method if I'm not mistaken

Reply:
Thank you again, Dante!